### PR TITLE
[1.9.4] Add drawFlowingWaterOverlay hook

### PR DESCRIPTION
**Pull request by gegy1000**
This PR adds a new hook, drawFlowingWaterOverlay(), which can be used to apply the same fix made to glass in 1.9 ([MC-35790](https://bugs.mojang.com/browse/MC-35790)) to custom mod blocks.

An example of usage (in custom block class): http://pastebin.com/3viMxS3U

### CODE REVIEW

**Comment by mezz** [comment]
I'm guessing there's no good way to detect this type of block without the event?
I took a quick look and didn't see a good way to differentiate a block like BlockPane from BlockGlass.

**Comment by blay09** [comment]
This looks like it should become a method in Block rather than an event.

File: patches/minecraft/net/minecraft/block/Block.java.patch [hunk 1]
```diff
@@ -1332,6 +1332,11 @@
 +    public void addInformation(ItemStack stack, EntityPlayer player, List<String> tooltip, boolean advanced)
 +    {
 +    }
++
```
Comment (shadowfacts) [inline patches/minecraft/net/minecraft/block/Block.java.patch]: JavaDoc?

**Comment by LexManos** [comment]
Don't we have a shouldSideBeRendered hook in Block already for things like stairs and stuff, this could be expanded for fluids no? @RainWarrior

**Comment by mezz** [comment]
Please update this PR to the 1.11 branch. 1.9.4 is not accepting PRs and 1.10 will soon be bugfix only.
@RainWarrior can you comment on this?

**Comment by mezz** [comment]
Any update @gegy1000 ?

**Comment by gegy1000** [comment]
@mezz Not yet, will be able to update next week.

**Pull request closed without merging**